modular_skyrat/modules/ices_events/code/events/ev_roleplay_check.dm

```diff
@@ -0,0 +1,10 @@
+/proc/engaged_role_play_check(mob/living/carbon/human/player)
+	var/turf/player_turf = get_turf(player)
+	var/area/player_area = get_area(player_turf)
+	if(!is_station_level(player_turf.z))
+		return TRUE
+
+	if(istype(player_area, /area/station/commons/dorms))
```

Comment:
Is dorms back in the protected spaces on-station? I thought we un-protected dorms in terms of "being on the station puts you at risk of un-targeted stuff"

Reply:
it's protected for like antags and disasters and stuff. but realistically if an erp'er gets appendicitis it gets ahealed. and disease outbreak is a waste of an event credit if it just infects someone locked in a dorm and also has to be manually rerolled

as discussed on discord we'll add args to determine what protections it checks

Reply:
👍